kuksa_databroker/databroker/src/main.rs

```diff
@@ -378,9 +378,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 );
             }
             (None, None) => {
-                return Err(
-                    "You must either provide TLS certificate and key or request insecure mode by --insecure.".into(),
-                );
+                warn!("Default behavior may change in the future, \
+                       please use --insecure if you want an insecure connection!");
```

Comment:
I think a little more context is needed around _what_ default behaviour may change in the future (as the user reading the log message doesn't have the context of the surrounding code here).
```suggestion
                warn!(
                    "Default behavior of accepting insecure connections \
                    when TLS is not configured may change in the future! \
                    Please use --insecure to explicitly enable this behaviour."
                );
```